samples/aws_lambda/lazy_aws_lambda_config.yaml

```diff
@@ -0,0 +1,40 @@
+region: us-east-1
+
+function_name: bolt_py_function
+handler: lazy_aws_lambda.handler
+description: My first lambda function
+runtime: python3.8
+# role: lambda_basic_execution
+role: bolt_python_lambda_invocation #  AWSLambdaFullAccess
```

Comment:
To use lazy listener functions feature, the IAM Role should have the permission to invoke Lambda functions.